src/demo.cmd.js

```diff
@@ -52,6 +40,21 @@ class InitCommand {
     this._type = 'simple';
   }
 
+  // eslint-disable-next-line class-methods-use-this
```

Comment:
I think there is no good reason to move the `execAsync` function into the class. The fact that you had to disable an ESLint check is a good warning sign. I think it has been caused largely by the need to easily stub the method, but given that creating real test conditions is easy, I'd go down that route. (see below)

Reply:
Good point! Hmmm, you're right. 

I'm torn, we can either keep the stub approach; or manipulate the environment by doing things like removing the .gitconfig in the before() and restoring it in the after(). 

If we move execAsync outside of the InitCommand class; I believe there's no way to stub it; such that the invocations inside of InitCommand are rerouted. 

@tripodsan and @trieloff what will be the choice? @tripdsan approach would keep us from doing things like moving;installing git as we can just stub the execAsync and make it return failure/success...for people who want to run the tests locally. 

Or we can use @trieloff approach; where we trade stubbing for actually creating the conditions, even when tests are local. 

Whatdya think fellas?